packages/@ember/-internals/views/lib/mixins/text_support.js

```diff
@@ -306,11 +307,19 @@ export default Mixin.create(TargetActionSupport, {
 // sendAction semantics for TextField are different from
 // the component semantics so this method normalizes them.
 function sendAction(eventName, view, event) {
-  let actionName = get(view, `attrs.${eventName}`) || get(view, eventName);
+  let action = get(view, `attrs.${eventName}`);
```

Comment:
The use of `actionName` for this was *super* confusing when reading the code, since it was only actually the *name* of an action *some* of the time. I opted to change it (back) to `action` to make it clear that it's *always* an action—sometimes as a string-named action, sometimes as a function.